src/apps/material/helper.ts

```diff
@@ -107,6 +107,23 @@ export const getManifestationLanguages = (manifestation: Manifestation) => {
   );
 };
 
+export const getManifestationLanguageIsoCode = (
+  manifestation: Manifestation[]
+) => {
+  const uniqueLanguages = uniqBy(
+    manifestation.map((m) => m.languages),
```

Comment:
```suggestion
  manifestations: Manifestation[]
) => {
  const uniqueLanguages = uniqBy(
    manifestations.map((m) => m.languages),
```

It is a bit weird that we have one helper function which accepts an array of manifestations while all the others only accept a single one. Anyhow: Here it makes sense.